app/controllers/home_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController
+  def index
+    @movies = Movie.order(upvotes: :desc).take(10)
```

Comment:
This could be a good place to use a model scope.

Example in controller:
Movie.best(10)

Example in the Movie model:
scope :best, -> (num) { order('upvotes DESC').limit(num)}